packages/turbo/ios/RNVisitableViewController.swift

```diff
@@ -26,11 +26,9 @@ class RNVisitableViewController: VisitableViewController {
   
   public var delegate: RNVisitableViewControllerDelegate?
   
-  // For native stack this function is called fon every screen change
-  // as the view is replaced in the view hierarchy every time we navigate to a screen
-  override func viewWillAppear(_ animated: Bool) {
-    super.viewWillAppear(animated)
-    delegate?.visitableWillAppear(visitable: self)
```

Comment:
We inherit from `VisitableViewController`, so calling `super.viewWillAppear(..)` will take care of notifying the `turboSession`.

We _also_ did that via the delegate call below, so the session would be notified twice leading to some funky behavior. We can safely remove this and let the parent notify the session.